src/main/java/thrift/logic/parser/ParserUtil.java

```diff
@@ -137,4 +141,26 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Checks if the currencies are valid
+     * @param currencies The currencies
```

Comment:
It should have an empty line between description and parameter section.

Reply:
Resolved.